firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/DevelopmentPlatformProvider.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.crashlytics.internal;
+
+import static com.google.firebase.crashlytics.internal.common.CommonUtils.getResourcesIdentifier;
+
+import android.content.Context;
+import androidx.annotation.Nullable;
+
+/** Provider for the development platform info. */
+public class DevelopmentPlatformProvider {
+  public static final String UNITY_PLATFORM = "Unity";
+
+  private static final String UNITY_EDITOR = "com.google.firebase.crashlytics.unity_version";
```

Comment:
Is there a reason this is called `UNITY_EDITOR` instead of something like `UNITY_VERSION_FIELD` ?

Reply:
That was the name in the original Unity version provider. But UNITY_VERSION_FIELD is better, renaming.